product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -6,85 +6,85 @@ deepToC: true
-- Double-check that both Postgres instances are not running.
-  - Use `pg_ctl` to stop both `/opt/postgres/dataold` and `/opt/postgres/data`
-- Check that you can upgrade safely.
-  - Use the `--check` option when running `bdr_pg_upgrade`
-- Run the Postgres major version upgrade.
+1. Double-check that both Postgres instances aren't running.
+  - Use `pg_ctl` to stop both `/opt/postgres/dataold` and `/opt/postgres/data`.
+1. Check that you can upgrade safely.
+  - Use the `--check` option when running `bdr_pg_upgrade`.
+1. Run the Postgres major version upgrade.
   - Run `bdr_pg_upgrade` again without the `--check` option.
-- Update the Postgres service configuration to the new version and refresh the system's service manager to apply the changes.
+1. Update the Postgres service configuration to the new version and refresh the system's service manager to apply the changes.
   - Replace the old version number with the new version number in the `postgres.service` file.
     - `sudo sed -i -e 's/<old_version_number>/<new_version_number>/g' /etc/systemd/system/postgres.service`
-- Start Postgres.
+1. Start Postgres.
   - `systemctl start postgres`
-- Validate the new Postgres version. 
-  - Connect to the bdrdb database and run `SELECT * FROM bdr.group_versions_details;`
-- Test the upgrade directly against the newly upgraded node.
-  - Run your test suite against while still SSH'd into the newly upgraded node.
-- Cleanup post-upgrade. 
+1. Validate the new Postgres version. 
+  - Connect to the `bdrdb` database and run `SELECT * FROM bdr.group_versions_details;`.
+1. Test the upgrade directly against the newly upgraded node.
+  - Run your test suite again while still connected by way of SSH into the newly upgraded node.
+1. Clean up post-upgrade. 
   - Run the new PG version's `vacuumdb`.
   - Remove the old version's data directory, `/opt/postgres/dataold`.
 
 ### Reconcile the upgrade with TPA
 
 TPA needs to continue to manage the deployment effectively after the upgrade. Therefore, it's necessary to reconcile the upgraded nodes with TPA. Follow these steps to update the configuration and redeploy the PGD cluster through TPA.
 
-- Change the config.yml of the TPA-managed cluster to the new version.
+- Change the `config.yml` of the TPA-managed cluster to the new version.
   - `cluster_vars: postgres_version: '<new_version_number>'`
-- Use `tpaexec` to redeploy the PGD cluster with the updated config.yml.
+- Use `tpaexec` to redeploy the PGD cluster with the updated `config.yml`.
   - `tpaexec deploy <cluster_name>`
 
-We will look at these steps in more detail below, specifically for upgrading a [shadow node](https://www.enterprisedb.com/docs/pgd/latest/terminology/#write-leader) from Postgres 15 to Postgres 16. 
+Details of these steps are shown in the worked example, specifically for upgrading a [shadow node](https://www.enterprisedb.com/docs/pgd/latest/terminology/#write-leader) from Postgres 15 to Postgres 16. 
```

Comment:
We don't write in first person but anyhow this text uses a mix. Let's stick with second person.